tar/test/test_copy.c

```diff
@@ -315,6 +315,19 @@ copy_basic(void)
 	assertEmptyFile("unpack.err");
 	assertEmptyFile("unpack.out");
 
+	verify_tree(LIMIT_NONE);
+
+	/*
+	 * Unpack a second time to make sure that things are still ok
+	 */
+	r = systemf("%s xf archive >unpack.out 2>unpack.err", testprog);
+	failure("Error invoking %s xf archive", testprog);
+	assertEqualInt(r, 0);
+
+	/* Verify that nothing went to stdout or stderr. */
+	assertEmptyFile("unpack.err");
+	assertEmptyFile("unpack.out");
+
```

Comment:
I hope this got in accidentially. I have added a test for SAFE_WRITES already and this one extracts with default flags. Please remove.